Rivet/Functions/Merge-Migration.ps1

```diff
@@ -306,6 +313,24 @@ function Merge-Migration
                     }
                 }
 
+                if( $op -is [Rivet.Operations.AddRowGuidColOperation] -or $op -is [Rivet.Operations.RemoveRowGuidColOperation] )
+                {
+                    $isRowGuidCol = ($op -is [Rivet.Operations.AddRowGuidColOperation])

```

Comment:
Nit: This variable seems unnecessary. You could do this check directly on line 326:
```
$tableOp.Columns[$columnIdx].RowGuidCol = ($op -is [Rivet.Operations.AddRowGuidColOperation])
```